docs/administration-guide/13-embedding.md

```diff
@@ -39,19 +40,19 @@ Then select "Embed this question in an application"
 
 ![Enable sharing for a question](images/embedding/03-enable-question.png)
 
-Here you will see a preview of the question or dashboard as it will appear in your application, as well as a panel that whos you the code you will need to insert in your application. 
+Here you will see a preview of the question or dashboard as it will appear in your application, as well as a panel that shows you the code you will need to insert in your application. 
 
 ![Preview](images/embedding/04-preview.png)
 
-Importantly, you will need to hit "Publish" when you first set up a  chart or dashboard for embedding and each time you change it. Also, any changes you make to the resource might require you to update the code in your own application to the latest code sample in the "Code Pane". 
+Importantly, you will need to hit "Publish" when you first set up a  chart or dashboard for embedding and each time you embedding settings. Also, any changes you make to the resource might require you to update the code in your own application to the latest code sample in the "Code Pane". 
```

Comment:
`each time you embedding settings`
should be
`each time you change your embedding settings`